gen-cert

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
```

Comment:
If we'll leave this for future debugging needs, it'd be nice to have a reminder on the context:  when and how to use.